x-pack/packages/security/authorization_core/src/privileges/privileges.test.ts

```diff
@@ -615,10 +616,12 @@ describe('features', () => {
         const actual = privileges.get();
         expect(actual).toHaveProperty(`${group}.all`, [
           actions.login,
-          ...(expectDecryptedTelemetry ? [actions.api.get('decryptedTelemetry')] : []),
-          ...(expectGetFeatures ? [actions.api.get('features')] : []),
-          ...(expectGetFeatures ? [actions.api.get('taskManager')] : []),
-          ...(expectGetFeatures ? [actions.api.get('manageSpaces')] : []),
+          ...(expectDecryptedTelemetry
+            ? [actions.api.get(ApiOperation.Read, 'decryptedTelemetry')]
+            : []),
+          ...(expectGetFeatures ? [actions.api.get(ApiOperation.Read, 'features')] : []),
+          ...(expectGetFeatures ? [actions.api.get(ApiOperation.Manage, 'taskManager')] : []),
+          ...(expectGetFeatures ? [actions.api.get(ApiOperation.Manage, 'manageSpaces')] : []),
```

Comment:
Maybe we can just make it, so it won't have double `manage`?
```suggestion
          ...(expectGetFeatures ? [actions.api.get(ApiOperation.Manage, 'spaces')] : []),
```